docs/detections/api/exceptions/exceptions-api-overview.asciidoc

```diff
@@ -37,19 +37,13 @@ entities:
 
 image::images/exceptions-logic.png[]
 
-IMPORTANT: Before you can create exceptions, you must create `.lists` and
+[float]
+=== Exceptions prerequisites
+
+Before you can create exceptions, you must create `.lists` and
```

Comment:
is this necessary? It's possible to create exception without using lists.

Just tested for user on space which does not have lists data streams.

Maybe we should add note

> Before you can create exceptions that uses lists, you must create `.lists` and

Reply:
I'm not sure if the .lists` and `.items` data streams are requirements for creating exceptions via the exceptions API. I was mainly reformatting the [information](https://www.elastic.co/guide/en/security/master/exceptions-api-overview.html#_kibana_role_requirements_2) that's currently in the docs.

RE your suggestion:
Are exceptions that use lists essentially [shared exception lists](https://www.elastic.co/guide/en/security/master/shared-exception-lists.html)? If so, I can update the language to reflect that.

Reply:
> Are exceptions that use lists essentially [shared exception lists](https://www.elastic.co/guide/en/security/master/shared-exception-lists.html)? If so, I can update the language to reflect that.

No, they are just sharable exception lists. As per https://www.elastic.co/guide/en/security/master/shared-exception-lists.html#add-exception-items, 3.b.

> Operator: Select an operator to define the condition:
    is | is not — Must be an exact match of the defined value.
    is one of | is not one of — Matches any of the defined values.
    exists | does not exist — The field exists.
    is in list | is not in list — Matches values in a value list.

Reply:
Gotcha, ok. So if I'm understanding correctly, there are _no list-related privs required for creating exceptions via the exceptions API_. However, if users want to [create shareable exception lists](https://www.elastic.co/guide/en/security/master/exceptions-api-create-shared-exception-list.html), they need to create the `.lists` and `.items` data streams and require privs to create those data streams and manage lists.

If the above is true, would it make sense to [this information](https://www.elastic.co/guide/en/security/master/exceptions-api-overview.html#_kibana_role_requirements_2) from the exceptions overview page and move it to the [page](https://www.elastic.co/guide/en/security/master/exceptions-api-create-shared-exception-list.html) that describes how to create shared exception lists via the API? Are are there any other places in the exception API docs that require this information?

Reply:
I don't think .lists and .items privs required to create shared exceptions lists or exception item or exceptions, **unless**
operator
> is in list | is not in list — Matches values in a value list.
need to be used  in it.

For simplicity, we may leave that both of these data streams needed, but in fact they are not required.

Reply:
Sorry for the late reply, @vitaliidm and thanks for clarifying. I think I understand now. If users are creating regular exceptions or shared exceptions that use value lists, they need to first create the `.lists` and `.items` data streams, and then they need role access to these data streams.

Reply:
Addressed this in [699b2a4](https://github.com/elastic/security-docs/pull/4524/commits/699b2a476a6536fb845ce5658467d350b76a5a8f).